openpype/hosts/maya/plugins/create/create_unreal_yeticache.py

```diff
@@ -0,0 +1,39 @@
+from openpype.hosts.maya.api import (
+    lib,
+    plugin
+)
+from openpype.lib import NumberDef
+
+
+class CreateYetiCache(plugin.MayaCreator):
+    """Output for procedural plugin nodes of Yeti """
+
+    identifier = "io.openpype.creators.maya.unrealyeticache"
+    label = "Unreal - Yeti Cache"
+    family = "yeticacheUE"
+    icon = "pagelines"
+
+    def get_instance_attr_defs(self):
+
+        defs = [
+            NumberDef("preroll",
+                      label="Preroll",
+                      minimum=0,
+                      default=0,
+                      decimals=0)
+        ]
+
+        # Add animation data without step and handles
+        defs.extend(lib.collect_animation_defs())
```

Comment:
Couldn't this be added when we initialize `defs` ?:
```py
defs = [
	NumberDef("preroll",
		label="Preroll",
		minimum=0,
		default=0,
		decimals=0),
	lib.collect_animation_defs()
]

Reply:
Technically yes, but in the lines below we're removing a few keys from that, so I think it makes the code more readable this way. What do you think?

Reply:
If it's for readability it's fine then; but by the time we do the removal they are already merged, it's just happens in two steps instead of one, or I got it wrong?